README.md

```diff
@@ -123,6 +123,7 @@ Spiderable.debug = true
  - Create template which you prefer to return, when page is not found
  - Set `Spiderable.customQuery`
  - Set iron router's `notFoundTemplate`
+ - Include a comment '<!-- response:status-code=404 -->' on your template. This way, we can ensure spiderable sends a 404 status code in the response headers
```

Comment:
@jazeee Currently as replacement of prev.version - no issues found. If I find any further issues - I'll send a new PR.